src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/chip.scss

```diff
@@ -19,6 +19,10 @@ $chipDisabledBackgroundColor: $silver;
     .icon {
         margin-left: 10px;
     }
+
+    > * {
```

Comment:
It is required because a `Loader` is passed to it right next to a text in the [`FieldFilterItem`](https://github.com/sulu/sulu/pull/5035/files#diff-1c4849a1f68379b37193d70c99797491R122). The `Loader` is a block element, so if I don't set this for it, it will cause the loader to appear in an own row.

IMO it is ok this way, because I think the Chip should never be stretched over multiple lines, and this statement helps with that.